object-construction-checker/tests/disabledframeworks/README.md

```diff
@@ -0,0 +1,2 @@
+These two test cases are adapted from _autovalue/UseAutoValueBuilder.java_ and _lombok/LombokBuilderExample.java_
+but removed all `// :: error:` comments.
```

Comment:
Can you add a sentence here with the reasoning? That is, this README should say that these are testcases for disabling framework support explicitly.